### ISSUE TITLE
Complete shaders following reference implementation

### ISSUE THREAD

**Issue by sinkingsugar** [fragcolor-xyz/chainblocks#141]
Following this
https://github.com/KhronosGroup/glTF-Sample-Viewer/tree/master/source/Renderer/shaders
Implement PBR shaders, getting also vertex skinning ready to be used by Animation system.

Moreover we don't need a material system yet cos chains and blocks and scripts ARE material definitions if wanted.

**Comment by sinkingsugar** [comment]
Depends on #140

**Comment by guusw** [comment]
Closing in favor of #170 based on the WebGPU backend
